### ISSUE TITLE
Proxy settings need hints

### ISSUE THREAD

**Issue by lyrl** [Blarc/ai-commits-intellij-plugin#34]
The place for setting up a proxy needs some hints, because now it is common to use a self-deployed reverse proxy server as an HTTP proxy, while the application actually sets up an HTTP/SOCKS proxy address.

**Comment by Blarc** [comment]
Hey. Thanks for opening this issue. Would you mind explaining me a bit how do you actually use this proxy? :pray:  I added this because someone else requested it (https://github.com/Blarc/ai-commits-intellij-plugin/issues/21), but I don't actually use it myself.

Even better, if you can prepare the hint message yourself :smiley: .

**Comment by lyrl** [comment]
Because ChatGTP is blocked in some countries, we need to use a proxy service. One of the reverse proxies used is Cloudflare's worker service, which runs a Node.js program on it to forward requests to OpenAI's API address. Below are the specific steps and sample code.
Proxy request to ChatGPT API, deployment steps:

1. Register and log in to your Cloudflare account
2. Create a new Cloudflare Worker
3. Copy and paste cloudflare-worker.js into the Cloudflare Worker editor
4. Save and deploy the Cloudflare Worker
5. In the Worker details page -> Trigger -> Custom Domains, add a custom domain for this worker.

repo: https://github.com/barretlee/cloudflare-proxy

code block: cloudflare-worker.js
```js
addEventListener("fetch", (event) => {
  event.respondWith(handleRequest(event.request));
});

async function handleRequest(request) {
  const url = new URL(request.url);
  const fetchAPI = request.url.replace(url.host, 'api.openai.com');
  
  // 部分代理工具，请求由浏览器发起，跨域请求时会先发送一个 preflight 进行检查，也就是 OPTIONS 请求
  // 需要响应该请求，否则后续的 POST 会失败
  const corsHeaders = {
    'Access-Control-Allow-Origin': '*',
    'Access-Control-Allow-Methods': 'OPTIONS',
    'Access-Control-Allow-Headers': '*',
  };
  if (request.method === 'OPTIONS') return new Response(null, { headers: corsHeaders });

  let body;
  if (request.method === 'POST') body = await request.json();

  const authKey = request.headers.get('Authorization');
  if (!authKey) return new Response("Not allowed", { status: 403 });
  
  const payload = {
    method: request.method,
    headers: {
      "Content-Type": "application/json",
      Authorization: authKey,
    },
    body: typeof body === 'object' ? JSON.stringify(body) : '{}',
  };
  // 在 Cloudflare 中，HEAD 和 GET 请求带 body 会报错
  if (['HEAD', 'GET'].includes(request.method)) delete payload.body;

  // 入参中如果包含了 stream=true，则表现形式为流式输出
  const response = await fetch(fetchAPI, payload);
  if (body && body.stream !== true) {
    const results = await response.json();
    return new Response(JSON.stringify(results), {
      status: response.status,
      headers: {
        "Content-Type": "application/json",
      },
    });
  } else {
    return new Response(response.body, {
      status: response.status,
      statusText: response.statusText,
      headers: response.headers,
    });
  }
}```

**Comment by Blarc** [comment]
I have released a new version [1.0.0](https://github.com/Blarc/ai-commits-intellij-plugin/releases/tag/v1.0.0) that contains the hint. 

Thank you for the the issue.
